.github/CODEOWNERS

```
@@ -1 +1 @@
-* @siboehm
\ No newline at end of file
+* @n-gao @siboehm
```

```suggestion
* @siboehm
```
let's do this in a different PR

---

actually I already did it in #12, see also https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-the-maintainer-list